circle.yml

```diff
@@ -0,0 +1,30 @@
+machine:
+  services:
+    - docker
+  environment:
+    BUILD_HOST: circle-build01.fhcrc.org
+
+dependencies:
+  override:
+    - pip install awscli
+
+test:
+  override:
+    - echo we need real unit tests here
+
+deployment:
+  prod:
+    branch: master
+    # branch: experiment/ci-cd
+    commands:
+      - aws s3 cp s3://fh-ctr-mofuz-test/motuz-cicd/id_rsa .
+      - chmod 0400 ./id_rsa
+      - scp -o StrictHostKeyChecking=no -i ./id_rsa bin/prod/update.sh ${REMOTE_USER}@${REMOTE_HOST}:~
+      - ssh -o StrictHostKeyChecking=no -i ./id_rsa ${REMOTE_USER}@${REMOTE_HOST} ./update.sh
+      - rm -f ./id_rsa
+
+general:
+  branches:
+    only:
+     - master
+    #  - experiment/ci-cd
```

Comment:
Remove

Reply:
See above comment, the same applies here.